tests/neo4j/shared.py

```diff
@@ -149,13 +149,7 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def requires_min_bolt_version(feature):
-    if not isinstance(feature, protocol.Feature):
-        raise TypeError("The arguments must be instances of Feature")
-    if not feature.name.startswith("BOLT_"):
-        raise ValueError("Bolt version feature expected")
-
-    min_version = feature.value.split(":")[-1]
+def requires_min_bolt_version(min_version):
```

Comment:
I'm not sure about this change. I think having a string instead of a feature flag as argument makes it easier to screw up by accidentally providing an invalid bolt version, doesn't it?

Reply:
The problem is that I wanted to get rid of the feature flag for Bolt 4.0 so that drivers cannot use it anymore.
Re-introducing the feature flag just for this function seemed a bit overkill.

Reply:
But why not mark the test as 4.1 minimum? We wouldn't test 4.0 anyway?

Reply:
It would work indeed. The reason I did not initially do it is because I also see this as a self-documenting feature. `requires_min_bolt_version` would give the first version a feature was introduced.
We can of course ditch that aspect completely and just use the next available feature flag.

Reply:
:thinking: that's good point. Let's leave it like this and hope it does not get removed once we drop support for all server versions that don't support a given feature.